core-library/src/main/java/org/silverpeas/core/admin/domain/driver/ldapdriver/LDAPTimeStampNDS.java

```diff
@@ -41,6 +42,12 @@ public int compareTo(Object other) {
     return timeStamp.compareTo(((LDAPTimeStampNDS) other).timeStamp);
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) This class overrides "hashCode()" and should therefore also override "equals()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1206)